coalib/settings/ConfigurationGathering.py

```diff
@@ -492,8 +500,12 @@ def gather_configuration(acquire_settings,
     aspectize_sections(sections)
     local_bears, global_bears = fill_settings(sections,
                                               acquire_settings)
+    only_global_bears = True;
+    for sec, value in local_bears.items():
+        if value:
+            only_global_bears = False;
     save_sections(sections)
-    warn_nonexistent_targets(targets, sections)
+    warn_nonexistent_targets(targets, sections, only_global_bears)
```

Comment:
I think you should add global bears option to `gather_configuration` function too, and then see build log, they fail because of this.

Reply:
what is meant by global bears option?

Reply:
the checks are failing because of the test cases. all bears till now have to have a `--files` tag. now I am changing that so the test yell at me why didn't coala quit when the `--files` was not specified